denlillemand/week4/ass5/app/components/namedlist.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import Message from './message.jsx';
+import { swapMessage, createMessage, deleteMessage, archiveMessage } from '../actioncreators/listactions.js';
+
+export default class NamedList extends React.Component {
+    constructor(props, context) {
+        super(props, context);
+        this.state = {
```

Comment:
Setting a full (blank) state is not required, but IMO it makes the code more readable as you have have stated what you expect to find in the state.

Just keep in mind to update the initial state definition if further members are added.